dtfs-central-api/src/helpers/get-fee-record-payment-entity-groups.ts

```diff
@@ -20,6 +15,15 @@ export const getFeeRecordPaymentEntityGroupsFromFeeRecordEntities = (feeRecords:
     }
   }
 
+  const getPaymentIdKeyFromPaymentEntities = (payments: PaymentEntity[]) => {
```

Comment:
Can this be moved outside of the overarching `getFeeRecordPaymentEntityGroups` function in this file please, then we can add a unit test